File: pyround1/1351.统计有序矩阵中的负数.py
#
# @lc app=leetcode.cn id=1351 lang=python3
#
# [1351] 统计有序矩阵中的负数
#

# @lc code=start
class Solution:
    def countNegatives(self, grid: List[List[int]]) -> int:
        count = 0
        for i in range(len(grid)):
            for j in range(len(grid[i])):
                if grid[i][j] < 0:
                    count += 1
        return count
# @lc code=end

File: 1351.统计有序矩阵中的负数.cpp
/*
 * @lc app=leetcode.cn id=1351 lang=cpp
 *
 * [1351] 统计有序矩阵中的负数
 */

// @lc code=start
class Solution {
public:
    int countNegatives(vector<vector<int>>& grid) {
        int count = 0;
        for(auto vec : grid)
        {
            for(auto ch : vec)
            {
                if(ch < 0)
                    count++;
            }
        }
        return count;
    }
};
// @lc code=end

